packages/code-du-travail-frontend/pages/kali.js

```diff
@@ -50,8 +49,8 @@ class Kali extends React.Component {
           title={title}
           emptyMessage="Cette convention collective n'a pas été trouvée"
           footer="Informations fournies par la DILA"
-          sourceType="Convention collective"
-          icon={ArticleIcon}
+          icon={null}
```

Comment:
why not removing the icon prop ?

Reply:
it's because there is a default value here https://github.com/SocialGouv/code-du-travail-numerique/blob/b4ff54cbf17f7093881536c8789d91b0cc453afe/packages/code-du-travail-frontend/src/common/Article.js#L12 so I have to explicitly say I don't want this default value